src/test/scala/nl/knaw/dans/easy/multideposit/CustomMatchers.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
```

Comment:
Okay, can't argue with the manual, I guess. Then we'll have to correct a number of poms ...